python/microns-morphology-api/setup.py

```diff
@@ -8,9 +8,10 @@
     exec(f.read())
 
 setup(
-    name="microns-morphology-config",
+    name="microns-morphology-api",
     version=__version__,
-    description="configuration for microns-morphology",
+    description="api for microns-morphology",
     author="Brendan Celii, Christos Papadopoulos",
-    packages=find_packages()
+    packages=find_packages(),
+    install_requires = ['trimesh==3.6.15']
```

Comment:
Noting how trimesh releases a patch for every merged PR, for now it could be fine to leave this as a fully pinned version until we can take another look at it.